Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -751,22 +761,66 @@ def check_name_entries_symbol_substitutions(fb, ttFont):
     fb.ok("No need to substitute copyright, registered and"
           " trademark symbols in name table entries of this font.")
 
-# DEPRECATED: 021 - "Checking fsSelection REGULAR bit"
-#             025 - "Checking fsSelection ITALIC bit"
-#             027 - "Checking fsSelection BOLD bit"
-#
-# Replaced by 129 - "Checking OS/2.fsSelection value"
 
-# DEPRECATED: 022 - "Checking that italicAngle <= 0"
-#             023 - "Checking that italicAngle is less than 20 degrees"
-#             024 - "Checking if italicAngle matches font style"
-#
-# Replaced by 130 - "Checking post.italicAngle value"
+@register_condition
+@condition
+def style(font):
+  """Determine font style from canonical filename."""
+  filename = os.path.split(font)[-1]
+  if '-' in filename:
+    return os.path.splitext(filename)[0].split('-')[1]
+
+
+@register_test
+@old_style_test(
+    id='com.google.fonts/test/020'
+  , conditions=['style']
```

Comment:
`if style is None`, this is skipped. Do we need (or do we have already) a check that fails if there's no `style` in the file name?